app/controllers/contentTypes.js

```diff
@@ -0,0 +1,7 @@
+var contentTypesHelper = require("../helpers/contentTypes");
+
+module.exports.getTypes = function(req, res) {
+	res.status(200).json(contentTypesHelper.toList().filter(function(type) {
+		return type.label !== "product_doc_version";
```

Comment:
ik haal die contenttypes op uit mongo en heb de product_doc_version daar nodig, maar hier gaat het om een platte array lijst om te gebruiken in de ui waar die doc version niet mag staan. Kan mss beter opgelost worden hoor, maar mongo wordt niet meer aangesproken hier